### PR TITLE
Add "Bibimbap" Theme

### PR DESCRIPTION
**Pull request by madhusudhand**
#### Changes proposed in this Pull Request:

This PR adds a new theme Bibimbap built by @beafialho 

Demo Site: https://bibimbap.mystagingwebsite.com
Designs: [Figma](aMrTGfvXPJPRvqJ9g6YuaT-fi-196%3A320)

#### Related issue(s):

### CODE REVIEW

File: bibimbap/readme.txt [hunk 1]
```diff
@@ -0,0 +1,31 @@
+=== Bibimbap ===
+Contributors: Automattic
+Requires at least: 5.8
+Tested up to: 5.9
```
Comment (mikachan) [inline bibimbap/readme.txt]: This can probably be changed to 6.1.1.

File: bibimbap/style.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
+/*
+Theme Name: Bibimbap
+Theme URI: 
+Author: Automattic
+Author URI: 
+Description:
```
Comment (mikachan) [inline bibimbap/style.css]:
```suggestion
Author URI: https://automattic.com
Description: A simple and fun restaurant theme.
```

File: bibimbap/style.css [hunk 3]
```diff
@@ -0,0 +1,66 @@
+/*
+Theme Name: Bibimbap
+Theme URI: 
+Author: Automattic
+Author URI: 
+Description: 
+Requires at least: 5.8
+Tested up to: 5.9
```
Comment (mikachan) [inline bibimbap/style.css]:
```suggestion
Tested up to: 6.1.1
```

File: bibimbap/style.css [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+ * theme support (block css) thus the following needs to be included
+ * since 'wp-block-styles' aren't used for this theme.
+ * https://github.com/WordPress/gutenberg/issues/45065
+ */
+.wp-block-table thead {
+	border-bottom: 3px solid;
+}
+.wp-block-table tfoot {
+	border-top: 3px solid;
+}
```
Comment (mikachan) [inline bibimbap/style.css]: These border styles are now in Gutenberg, so we don't need to include them in the CSS.

File: bibimbap/templates/404.html [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+<!-- wp:group {"align":"wide","layout":{"type":"default"}} -->
+<div class="wp-block-group alignwide"><!-- wp:paragraph -->
+<p>It looks like nothing was found at this location. Maybe try a search?</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:search {"label":"","showLabel":false,"style":{"border":{"radius":"2px"}}} /--></div>
+<!-- /wp:group --></main>
+<!-- /wp:group -->
+
+<!-- wp:template-part {"slug":"footer","tagName":"footer"} /-->
```
Comment (mikachan) [inline bibimbap/templates/404.html:5]: Should this file be using the 404 pattern?

File: bibimbap/templates/index.html [hunk 6]
```diff
@@ -0,0 +1,33 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+
+<!-- wp:query {"queryId":15,"query":{"perPage":6,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"exclude","inherit":true},"displayLayout":{"type":"flex","columns":3},"layout":{"type":"constrained"}} -->
```
Comment (mikachan) [inline bibimbap/templates/index.html]:
```suggestion
<!-- wp:query {"query":{"perPage":6,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"exclude","inherit":true},"displayLayout":{"type":"flex","columns":3},"layout":{"type":"constrained"}} -->
```

File: bibimbap/templates/search.html [hunk 7]
```diff
@@ -0,0 +1,35 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+
+<!-- wp:query {"queryId":15,"query":{"perPage":6,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"exclude","inherit":true},"displayLayout":{"type":"flex","columns":3},"layout":{"type":"constrained"}} -->
```
Comment (mikachan) [inline bibimbap/templates/search.html]:
```suggestion
<!-- wp:query {"query":{"perPage":6,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"exclude","inherit":true},"displayLayout":{"type":"flex","columns":3},"layout":{"type":"constrained"}} -->
```

File: bibimbap/templates/archive.html [hunk 8]
```diff
@@ -0,0 +1,31 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+
+<!-- wp:query {"queryId":15,"query":{"perPage":6,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"exclude","inherit":true},"displayLayout":{"type":"flex","columns":3},"layout":{"type":"constrained"}} -->
```
Comment (mikachan) [inline bibimbap/templates/archive.html]:
```suggestion
<!-- wp:query {"query":{"perPage":6,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"exclude","inherit":true},"displayLayout":{"type":"flex","columns":3},"layout":{"type":"constrained"}} -->
```

File: bibimbap/theme.json [hunk 9]
```diff
@@ -0,0 +1,565 @@
...
+				},
+				{
+					"color": "#c74200",
+					"name": "Secondary",
+					"slug": "secondary"
+				},
+				{
+					"color": "#000000",
+					"name": "Foreground",
+					"slug": "foreground"
```
Comment (mikachan) [inline bibimbap/theme.json]: This is the same colour as Primary, so I don't think we need both of these definitions.

**Review by mikachan — COMMENTED** [review]
Looking good! I really like the Cooper Hewitt font here.

I noticed that some of the images are quite large, including the screenshot. I think it's worth compressing these so the file sizes are smaller.

File: bibimbap/readme.txt [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
```
Comment (madhusudhand) [inline bibimbap/readme.txt:31]: @beafialho could you provide image source for mention in readme.

File: bibimbap/templates/404.html (hunk 5 above)
Comment (madhusudhand) [reply bibimbap/templates/404.html:5]: Yeah. updated the template.

**Comment by madhusudhand** [comment]
Thanks @mikachan for the review. I have addressed all the comments.

**Review by mikachan — APPROVED** [review]

**Comment by mikachan** [comment]
I've just noticed that a lot of the images still have pretty large file sizes. I've opened a separate issue: https://github.com/Automattic/themes/issues/6811

**Pull request merged**